index.js

```diff
@@ -72,14 +73,16 @@ const resolver = Consumer.create({
         requestType = 'GET';
         const getCaseResponse = await getCase.fetch();
         const isCaseFound = (getCaseResponse.exists ? 'found' : 'not found');
-        logger.info({ externalID, message: 'Casework GET request successful. externalId:' +
-          `${externalID} was ${isCaseFound}`});
+        logger.info({
+          externalID, message: 'Casework GET request successful. externalId:' +
+            `${externalID} was ${isCaseFound}`
+        });
```

Comment:
I would use template literals
```suggestion
logger.info({
  message: `Casework GET request successful. External ID: ${externalID} was ${isCaseFound}`,
  externalID: externalID,
  status: isCaseFound
});

```
When you use a logger you can separate the fields (externalID and status) in the log object to make the logs more structured and easier to search/filter.